### PR TITLE
Docs: add hidden_users configuration field

### PR DESCRIPTION
**Pull request by AgnesToulet**
**What this PR does / why we need it**:
Add the `hidden_users` configuration field to the documentation

### CODE REVIEW

File: docs/sources/administration/configuration.md [hunk 1]
```diff
@@ -623,6 +623,10 @@ The duration in time a user invitation remains valid before expiring.
 This setting should be expressed as a duration. Examples: 6h (hours), 2d (days), 1w (week).
 Default is `24h` (24 hours). The minimum supported duration is `15m` (15 minutes).
 
+### hidden_users
+
+A comma-separated list of usernames to hide them in the Grafana UI. These users are shown to Grafana admins and to themselves.
```
Comment (achatterjee-grafana) [inline docs/sources/administration/configuration.md:628]:
```suggestion
A comma-separated list of usernames to hide them in the Grafana UI. They are also visible to Grafana administrators and to themselves.
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added a suggestion for improvement.

File: docs/sources/administration/configuration.md (hunk 1 above)
Comment (oddlittlebird) [reply docs/sources/administration/configuration.md:628]: The first sentence is missing a subject and a verb. WHAT is the comma-separated list doing? Are you adding a list of names to hide from the UI? If so, why? Do we explain that somewhere?

**Review by oddlittlebird — CHANGES_REQUESTED** [review]

File: docs/sources/administration/configuration.md [hunk 2]
```diff
@@ -623,6 +623,10 @@ The duration in time a user invitation remains valid before expiring.
 This setting should be expressed as a duration. Examples: 6h (hours), 2d (days), 1w (week).
 Default is `24h` (24 hours). The minimum supported duration is `15m` (15 minutes).
 
+### hidden_users
+
+Set a comma-separated list of usernames to hide the corresponding users in the Grafana UI. They are still visible to Grafana administrators and to themselves.
```
Comment (achatterjee-grafana) [inline docs/sources/administration/configuration.md:628]:
```suggestion
This is a comma-separated list of usernames. Users specified here are hidden in the Grafana UI. They are still visible to Grafana administrators and to themselves.
```

**Review by achatterjee-grafana — COMMENTED** [review]
Suggested rewording.

**Review by achatterjee-grafana — APPROVED** [review]
LGTM

File: docs/sources/whatsnew/whats-new-in-v7-4.md [hunk 3]
```diff
@@ -95,6 +95,11 @@ CSP support is disabled by default, to enable it you must set `content_security_
 
 You can lock down what can be done in the frontend code. Lock down what can be loaded, what JavaScript is executed. Not compatible with some plugins.
 
+### Hide users in UI
+
+You can now use the `hidden_users` configuration setting to hide specific users in the UI. With this feature, users created for automation purpose can never be shown in Grafana UI.
```
Comment (xlson) [inline docs/sources/whatsnew/whats-new-in-v7-4.md:100]:
```suggestion
You can now use the `hidden_users` configuration setting to hide specific users in the UI. For example, this feature can be used to hide users that are used for automation purposes.
```

Perhaps?

**Review by xlson — APPROVED** [review]

**Comment by AgnesToulet** [comment]
@oddlittlebird Can you take a final look please? As you requested changes before, I won't be able to merge until you update your review.

**Review by oddlittlebird — APPROVED** [review]
LGTM!

**Pull request merged**